package.json

```diff
@@ -12,9 +12,12 @@
     "npm": ">=6.4.1"
   },
   "dependencies": {
-    "@material-ui/core": "^4.11.3",
-    "@material-ui/icons": "^4.11.2",
-    "@material-ui/lab": "^4.0.0-alpha.57",
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
```

Comment:
Do we actually need "@emotion/react" and "@emtion/styled"? I know that MUI is using them under the hood, but we don't seem to be actually using them anywhere and it's better to keep the dep list as minimal as possible.

Reply:
Thank you so much @ibacher, for the review. I had not used emotion at the beginning but it ended up with some error **Unhandled Rejection error** in Reducer. And was fixed by adding emotion so had to add this dep.